dashboards/proposal.md

```diff
@@ -0,0 +1,74 @@
+# Proposal for Incubation
+
+## Subproject name
+
+Jupyter Dynamic Dashboards from Notebooks (jupyter-incubator/dashboards)
+
+## Development team and Advocate
+
+Subproject development team:
+
+* Peter Parente, IBM (`@parente`)
+* Gino Bustelo, IBM (`@ginobustelo`)
+* Justin Tyberg, IBM (`@jtyberg `)
+* Dan Gisolfi, IBM (`@vinomaster`)
+
+Steering Council Advocate:
+
+* Brian Granger (`@ellisonbg`)
```

Comment:
If you wish to, you can also make me an advocate on this one as it relates to #3.